codemon/CodemonListen.py

```diff
@@ -38,8 +38,8 @@ def isModified(event):
   if filename != foldername and filename != "prog" and filename != "input.txt": 
     print(colored.yellow('\nChange made at '+ filename))
     print(colored.cyan('\nCompiling '+ filename))
-    os.system('g++ ' + filename + ' -o ' + 'prog')
-    print('Running')	
-    print()
-    print(colored.yellow('Taking inputs from input.txt'))
-    os.system(f'{os.getcwd()}/prog < input.txt')
+    if os.system('g++ ' + filename + ' -o ' + 'prog') == 0:
+        print('Running')	
+        print()

```

Comment:
Could you get rid of this extra `print()` here ?

Reply:
I did not add that extra print, it was already there. I can remove it if you want?

Reply:
It was added from a previous PR, you may remove it.

Reply:
okay, will do.